### PR TITLE
@eessex => [Bugfix] Missing sidebar in <Article />

### PR DESCRIPTION
**Pull request by damassi**
Updates https://github.com/artsy/publishing/issues/91 along with some light cleanup.

### CODE REVIEW

**Comment by damassi** [comment]
@eessex / @kanaabe - Q: 

> Related article sidebar not showing (we should show related articles here but not the panel display units)

What are "panel display units"?

File: desktop/apps/article2/components/App.js [hunk 1]
```diff
@@ -1,12 +1,11 @@
...
+import ArticleModel from 'desktop/models/article.coffee'
 import EditButton from 'desktop/apps/article2/components/EditButton'
+import InfiniteScrollArticle from './InfiniteScrollArticle'
+import PropTypes from 'prop-types'
+import React from 'react'
 import SuperArticleView from 'desktop/components/article/client/super_article.coffee'
-import ArticleModel from 'desktop/models/article.coffee'
-const { Article } = components
+import { Article } from '@artsy/reaction-force/dist/Components/Publishing'
+import { data as sd } from 'sharify'
```
Comment (damassi) [inline desktop/apps/article2/components/App.js]: Alpha on imports (highlight "sort natural" or related in editor)  -- Order logic follows: 
```js
*
_ 
A
B
a
b
{ A }
{ B } 
{ a }
{ b }
```

**Comment by eessex** [comment]
@damassi display units are our new ad units, they come in two layouts -- panel (sidebar) and canvas (full bleed below an article)

File: desktop/apps/article2/components/App.js [hunk 2]
```diff
@@ -28,21 +27,29 @@ export default class App extends React.Component {
...
+      const headerHeight = `calc(100vh - ${navHeight})`
 
       return (
         <Article
           article={article}
-          headerHeight={`calc(100vh - ${navHeight})`}
-          marginTop={article.layout === 'standard' ? '100' : '0'}
+          relatedArticlesForPanel={article.relatedArticlesPanel}
+          relatedArticlesForCanvas={article.relatedArticlesCanvas}
+          headerHeight={headerHeight}
```
Comment (eessex) [inline desktop/apps/article2/components/App.js]: I think we might only want the panel articles here -- super-articles have their own footer with their sub-articles, and are excluded from infinite scroll. 
@owendodd do super-articles have the related canvas at the bottom?

**Comment by eessex** [comment]
Is a very big diff on the lock file, is this all from updating reaction?  If not maybe is safer to copy the existing master and only remove/add reaction.

**Comment by eessex** [comment]
This looks good to go pending question about related articles canvas.

File: desktop/apps/article2/components/App.js (hunk 2 above)
Comment (owendodd) [reply desktop/apps/article2/components/App.js]: @eessex no they shouldn't as they are almost always brand partnerships

Comment (owendodd) [reply desktop/apps/article2/components/App.js]: we'll have to revisit this in the future, but that's the most straightforward answer for the time being

**Comment by kanaabe** [comment]
Sorry it looks like my last PR caused all the conflicts! This looks good to me after we remove that `relatedArticlesForCanvas`. Thanks for the cleanup 😄

**Comment by damassi** [comment]
@eessex - Lockfile updates were from reaction. 

(Fixed conflicts, rebased and restarted due to weird test failures.)

**Comment by kanaabe** [comment]
Hm not sure why it's getting stuck in CI. Tried rebuilding and yarn registry status looks ok.

**Pull request merged**